### PR TITLE
Serialize labbooks (½): Handles a user directly copying an activity.

### PR DESCRIPTION
**Pull request by knowuh**

### CODE REVIEW

**Comment by knowuh** [comment]
Note:  This is based off labbook reporting branch. It will be easier to review once the other PR is merged.

File: app/services/page_serialization_helper.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class PageSerializationHelper
+  def chache_interactive_copy(original_interactive, copy_of_interactive)
```
Comment (scytacki) [inline app/services/page_serialization_helper.rb]: chache - cache

Comment (knowuh) [inline app/services/page_serialization_helper.rb]: Thanks for catching that. -- I will roll that into the next set of changes.

Comment (dougmartin) [inline app/services/page_serialization_helper.rb]: Noah loves Chache?

**Comment by scytacki** [comment]
This looks good to me. As we talked about, it might make sense in the future to add a more complex system for handling references when copying/import/export handling of references.

**Pull request merged**